modules/gles/Makefile.am

```diff
@@ -19,6 +19,7 @@ xcam_gles_sources =                  \
     gl_video_buffer.cpp              \
     gl_compute_program.cpp           \
     gl_command.cpp                   \
+    gl_utils.cpp                     \
```

Comment:
merge gl_utils and gl_error.h into one.
e.g. use gl_common.h or gl_utils.h as new file name.